src/store.js

```diff
@@ -36,17 +38,14 @@ store.dispatch(setContext('readmeContext', readmeContext));
 store.dispatch(setLessons(lessons));
 store.dispatch(setFilter(getTags(lessonContext, courseContext)));
 
-let initialMode = true;
-let initialLanguage = 'nb';
-let initialWelcomeBox = true;
+const initialMode = loadFromLocalStorage('isStudentMode', true);
+const initialWelcomeBox = loadFromLocalStorage('welcomeBox', true);
+let initialLanguage = loadFromLocalStorage('language', 'nb');
+if (localStorage.language) { initialLanguage = localStorage.language; }
 
-if (typeof localStorage !== 'undefined') {
-  if (localStorage.isStudentMode) { initialMode = JSON.parse(localStorage.isStudentMode); }
-  if (localStorage.language) { initialLanguage = localStorage.language; }
-  if (localStorage.welcomeBox) { initialWelcomeBox = JSON.parse(localStorage.welcomeBox); }
-}
 store.dispatch(setMode(initialMode));
-store.dispatch(setLanguage(initialLanguage));
 store.dispatch(setWelcomeBox(initialWelcomeBox));
+store.dispatch(setLanguage(initialLanguage));
+store.dispatch(setCheckboxes('', {}));
```

Comment:
Why are we setting an empty object in localstorage where the key is an empty string?